open_humans/templatetags/utilities.py

```diff
@@ -298,3 +324,37 @@ def add_string(a, b):
     Like `add` but coerces to strings instead of integers.
     """
     return str(a) + str(b)
+
+
+@register.tag
+def render_if_visible(parser, token):
+    """
+    Is a member publicly sharing data but wishes that membership to not be public?
+    """
+    try:
+        tag, user_t, project_t = token.split_contents()
```

Comment:
I think `user_t` is actually receiving a `member_t` and `project_t` might be more clearly named as a `source` or `source_label_t`? Naming should be consistent with the object being passed.